src/v0/destinations/braze/transform.js

```diff
@@ -401,6 +402,7 @@ function processTrackEvent(messageType, message, destination, mappingJson, proce
   payload.properties = properties;
 
   payload = setExternalIdOrAliasObject(payload, message);
+  payload = addAppId(payload, message);
   if (payload) {
```

Comment:
is it necessary?

Reply:
yes, this is what support they asked for

Reply:
I am asking about checking for payload being not defined. as in line 402 we are setting properties on payload